beacon-chain/state/fieldtrie/field_trie.go

```diff
@@ -1,65 +1,66 @@
-package v1
+package fieldtrie
 
 import (
 	"reflect"
 	"sync"
 
 	"github.com/pkg/errors"
 	"github.com/prysmaticlabs/prysm/beacon-chain/state/stateutil"
+	"github.com/prysmaticlabs/prysm/beacon-chain/state/types"
 )
 
 // FieldTrie is the representation of the representative
 // trie of the particular field.
 type FieldTrie struct {
 	*sync.RWMutex
-	reference   *stateutil.Reference
-	fieldLayers [][]*[32]byte
-	field       fieldIndex
+	Reference   *stateutil.Reference
```

Comment:
Do we really need to export this? How about providing getter and setter around it?